propel/propel/2017-02-14.yaml

```diff
@@ -0,0 +1,8 @@
+title:     SQL injection possible with limit() on MySQL
+link:      https://github.com/propelorm/Propel2/issues/1463
+cve:       ~
+branches:
+    "2.0":
+        time:     2018-02-16 13:38:00
+        versions: ['>=2.0.0-alpha1', '<3']
```

Comment:
this has actually been merged, so the upper bound should be `<= 2.0.0-alpha7`. If they do a new release of Propel 2, it will be unaffected.

Reply:
*if* - in that case, we'll update these definitions. For now, everything in the released versions is affected.

Reply:
thing is, the commit is already merged in Prople 2. So we know that 2.0.0-alpha7 will always be the last affected version (unless they do crazy stuff by tagging a release on an older commit before the security fix, but then they would be insane)

Reply:
> unless they do crazy stuff by tagging a release on an older commit before the security fix

which we won't.

Reply:
@marcj don't get me wrong, I have full trust in the maintainers' good faith, but the current knowledge is that no releases in the `2.x` range are unaffected, so this repository should simply reflect that. Range can be adapted after a release is there :+1:

Reply:
https://github.com/propelorm/Propel2/releases/tag/2.0.0-alpha8 has been tagged 1minute before you posted it. :)

Reply:
:+1: awesome, then this can be changed